release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -557,6 +557,13 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-expose-mtu"]
+==== Exposing MTU for vfio-pci SR-IOV devices
+
+With this release, MTU on virtual function using the vfio-pci driver is available in the network-status pod annotation, and inside the container. 
+
+For more information, see xref:../networking/hardware_networks/add-pod.html#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
```

Comment:
In any case, this xref cannot have html in it, it must point to an adoc file:
```suggestion
For more information, see xref:../networking/hardware_networks/add-pod.adoc#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
```

Reply:
Thanks @skopacz1 for your feedback. I have applied your suggestion and changed it to adoc.